src/lib_ccx/ccx_common_constants.h

```diff
@@ -248,19 +248,19 @@ typedef enum {
 
 enum ccx_code_type
 {
-	CCX_CODEC_ANY,
-	CCX_CODEC_TELETEXT,
-	CCX_CODEC_DVB,
-	CCX_CODEC_ISDB_CC,
-	CCX_CODEC_ATSC_CC,
-	CCX_CODEC_NONE,
+	CCX_CODEC_ANY = 0,
+	CCX_CODEC_TELETEXT = 1,
+	CCX_CODEC_DVB = 2,
+	CCX_CODEC_ISDB_CC = 3,
+	CCX_CODEC_ATSC_CC = 4,
+	CCX_CODEC_NONE = 5
 };
 
 enum cdp_section_type
 {
-	CDP_SECTION_DATA = 0x72,
-	CDP_SECTION_SVC_INFO = 0x73,
-	CDP_SECTION_FOOTER = 0x74
+	CDP_SECTION_DATA = 0x72, // If these numbers realy mean something
```

Comment:
They are defined in SMPTE 334. 
We don't really want to duplicate the standard documents in our code :-)

Reply:
Could not find it anywhere for free o_O. Do you have a link to it?